rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/start_scan/action.py

```diff
@@ -0,0 +1,60 @@
+import insightconnect_plugin_runtime
+from .schema import StartScanInput, StartScanOutput, Input, Component
+
+# Custom imports below
+from insightconnect_plugin_runtime.exceptions import PluginException
+from ipaddress import ip_address, IPv4Address, IPv6Address
+
+
+def format_body(hostnames: str, ips: str, vulns: str):
```

Comment:
Since this is used by both asset search and start scan can we add it to the util package?

Reply:
Nevermind, see comment in asset search